### PR TITLE
Upgrade to Testcontainers 1.14.0

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

File: dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/FATSuite.java [hunk 1]
```diff
@@ -30,83 +30,83 @@
...
-            ps.executeUpdate();
-
-            //Close statements
-            ps.close();
-            stmt.close();
-        }
-    }
+	// TODO replace this container with the official oracle-xe container if/when it is available without a license
+	static OracleContainer oracle = new OracleContainer("kyleaure/oracle-18.4.0-xe-prebuilt:1.0")
+			.withExposedPorts(1521, 8080) // need to manually expose ports due to regression in 1.14.0
```
Comment (KyleAure) [inline dev/com.ibm.ws.jdbc_fat_oracle/fat/src/com/ibm/ws/jdbc/fat/oracle/FATSuite.java:38]: The oracle container uses ports 1521, and 5500.
This regression will also affect fattest.databases.  You'll need to expose these ports here as well: 
https://github.com/OpenLiberty/open-liberty/blob/dfd64634f4f0715db178b558e88f52c9b74c8f05/dev/fattest.databases/src/componenttest/topology/database/container/DatabaseContainerFactory.java#L122-L124

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#run-libby-bot

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Pull request merged**